Grains/dalle-image-generator/MultiImageGeneratorGrain.cs

```diff
@@ -43,6 +44,11 @@ public async Task<MultiImageGenerationGrainResponse> GenerateMultipleImagesAsync
 
             _multiImageGenerationState.State.Prompt = prompt;
             _multiImageGenerationState.State.Traits = traits;
+            var schedulerGrain = GrainFactory.GetGrain<ISchedulerGrain>("SchedulerGrain");
+
+            //get timestamp
+            var requestTimestamp = DateTime.UtcNow;
+            long unixTimestamp = ((DateTimeOffset)requestTimestamp).ToUnixTimeSeconds();
```

Comment:
@argfoo-nft pls check the timestamp generation logic
this uses utc time
is this correct approach/

Reply:
yes, this is correct approach